index.html

```diff
@@ -35,6 +35,136 @@ <h1>I'm Yohannes</h1>
+        </div>
+        <div class="art-item">
+            <h2 class="data">Professional Art<br>
+                Printing Data </h2>
+             <p class="selections"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+             <ul class="buttons">
+                <li>html</li> 
+                <li>bootstrap</li> 
+                <li>Ruby</li>
+               </ul>
+             <p class="invite">See Project</p>
+        </div>
+        <div class="art-item">
+            <h2 class="data">Professional Art<br>
+                Printing Data </h2>
+             <p class="selections"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+             <ul class="buttons">
+                <li>html</li> 
+                <li>bootstrap</li> 
+                <li>Ruby</li>
+               </ul>
+             <p class="invite">See Project</p>
+        </div>
+        <div class="art-item">
+            <h2 class="data">Professional Art<br>
+                Printing Data </h2>
+             <p class="selections"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+             <ul class="buttons">
+                <li>html</li> 
+                <li>bootstrap</li> 
+                <li>Ruby</li>
+               </ul>
+             <p class="invite">See Project</p>
+        </div>
+        <div class="art-item">
+            <h2 class="data">Professional Art<br>
+                Printing Data </h2>
+             <p class="selections"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+             <ul class="buttons">
+                <li>html</li> 
+                <li>bootstrap</li> 
+                <li>Ruby</li>
+               </ul>
+             <p class="invite">See Project</p>
+        </div>
+        <div class="art-item">
+            <h2 class="data">Professional Art<br>
+                Printing Data </h2>
+             <p class="selections"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+             <ul class="buttons">
+                <li>html</li> 
+                <li>bootstrap</li> 
+                <li>Ruby</li>
+               </ul>
+             <p class="invite">See Project</p>
+        </div>
+        
+        </div>
+        </div>
+
+    </section>
+    <section class="about-me">
+        <div class="myresume">
+        <h3 class='h3-title'>About me</h3>
+        <p class="about-paragraph"> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <button class="resume">Get My Resume</button>
```

Comment:
- [ ] [**OPTIONAL**] Please let's make this button more interactive by including button styling effects like `Hover`, `Pressed` and `Active`.